manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -153,6 +153,10 @@ spec:
             value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Release.Namespace }}.svc:15012
           - name: PILOT_ENABLE_ANALYSIS
             value: "{{ .Values.global.istiod.enableAnalysis }}"
+{{- if .Values.global.istiod.enableAnalysis }}
```

Comment:
Should not add a config here, users can set arbitrary env vars already (values.pilot.env.foo=bar)

Reply:
add it here so that user can do --set values.global.istiod.analysisInterval=30s from command line like they set enableAnalysis. If you @howardjohn , I can certainly remove that.

Reply:
I agree with John.  Users can already set arbitrary environment variables usnig the `values.pilot.env` section, so there's no need to build a dedicated values.yaml field for this.